01-intro.Rmd

```diff
@@ -0,0 +1,127 @@
+4. **Discuss and review code** - teams participate in code reviews by commenting, testing, and reviewing open pull requests. Code review is at the centre of an open and participatory culture. Pull requests should be approved by at least one collaborator - we recommend you assign a specific reviewer(s) to check the work. In order to ensure the master branch remains 'production ready' we also recommend that the master branch is protected and any changes are tested by the reviewer before approving.
+5. **Merge** - upon clicking merge, GitHub automatically performs the equivalent of a local `git merge` operation. GitHub also keeps the entire branch development history on the merged pull request.
+6. **Deploy** - teams can choose the best release cycles or incorporate continuous integration tools and operate with the assurance that code on the deployment branch has gone through a robust workflow.
+
+
+## Setup {-}
+
+### Git {-}
+
+*These intructions should only need to be followed once or when setting up a new device.*
+
+1. **Install Git** - Git is a free, open-source, software available from https://git-scm.com/. If you work within NHS National Services Scotland, request the software from IT through the [Service Portal](https://nhsnss.service-now.com/sp). Once authorised, this will allow you to download it from the Software Center on your machine. 
+
+This will install three applications on your computer, go to Start > Git (folder):
+
+* **Git Bash** - this is the command line (Shell) interface for git. This will allow you to enter git commands, i.e. `git init`, and can be launched by right clicking in a directory/folder and selecting "Git Bash Here".
+* **Git CMD** - this is a deprecated version, similar to the Bash interface, and shouldn't be used here.
+* **Git GUI** - this is git's version of a Graphical User Interface. This will perform the same functions as "Git Bash" by the user selecting options rather than having to type in the commands. This guide will outline using integrations on software like RStudio to utilise GUIs rather than this option.
+
+2. **Configure your details** - Git needs to know who you are, use the following commands to configure your username and email (*if you're a GitHub user, use the same email and username you're registered with on GitHub.*):
+
+* `git config --global user.email <NHS email address>`
+* `git config --global user.name <your name>`
+
+At any time you can check what the current user details are using:
+
+* `git config --global user.email`
+
+
+### GitHub {-}
+
+These instructions are for first-time users of GitHub and highlight the steps in setting up a GitHub account.
+
+1. **Sign up** - In order to use GitHub, you'll need to have a personal GitHub account, you can set that up on the [GitHub website](http://github.com). If using your NHS email address ensure you **don't use the same password as your NHS email account**. 
```

Comment:
People also shouldn't be using their LDAP passwords, especially if their username is the same as their LDAP login.